pyproject.toml

```diff
@@ -1,25 +1,111 @@
 [build-system]
 requires = [
-    "setuptools",
-    "wheel",
-    "numpy",
-    "ninja",
-    "pyyaml",
-    "setuptools",
-    "cffi",
-    "typing_extensions",
-    "future",
-    "tensorrt >=8.6,<8.7"
+    "setuptools==68.0.0",
+    "packaging>=23.1",
+    "wheel>=0.40.0",
+    "numpy>=1.25.0",
+    "ninja>=1.11.0",
+    "pyyaml>=6.0",
+    "cffi>=1.15.1",
+    "typing_extensions>=4.7.0",
+    "future>=0.18.3",
+    "tensorrt>=8.6,<8.7",
+    #"torch >=2.0.0,<2.1.0",
+    "torch==2.1.0.dev20230619+cu121",
```

Comment:
This distribution could not be found when building locally - is there a way to specify extra build indices?

Reply:
You would do this from the commandline